tests/unit/test_metrics.py

```diff
@@ -0,0 +1,39 @@
+"""Testing functions of metrics module."""
+from datetime import datetime
+
+import numpy as np
+import polars as pl
+import pytest
+
+from wristpy.core.models import Measurement
+from wristpy.processing.metrics import euclidean_norm
+
+TEST_LENGTH = 100
+
+@pytest.mark.parametrize(
+        "x,y,z, expected_enmo",
+        [
+            (np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 0),
+            (np.ones(TEST_LENGTH), np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 0),
+            (np.ones(TEST_LENGTH)*2, np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 1),
+            (np.ones(TEST_LENGTH)*3, np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 2)
```

Comment:
Love the parameterization; though try to limit it to unique cases. For example, I see three unique cases here that could be tested: 1) euclidean norm-1 is negative, so result is 0. 2) euclidean norm-1 is zero, result should still be 0, and 3) euclidean norm is positive. In at least one of these cases (probably the latter), try to use at least one test that has non-zero input for all three.

Reply:
Okay i'll add a case (should i add or just change the third case?) where I'll use a pythagorean quadruple 1,2,2,3. That does make me wonder, although this is still pretty easy to verify, do I have to at some point "show my work" when coming up with ground truth values that are based off of calculations like this ?

Reply:
I wouldn't expect you to show this in the documentation (I'd expect a reader of the code to be able to look up a euclidean norm algorithm themselves), that being said, you need to be able to verify the correctness of any algorithm that you implement. This can be done by hand (relatively straightforward here, see below for [1,2,2,3]), or for more complex algorithms by verifying against known cases and/or other implementations. 
 
```
sqrt(1^2 + 2^2 + 2^2 + 3^2) - 1
= sqrt(1+4+4+9) - 1
= sqrt(18) - 1
= sqrt(9) * sqrt(2) - 1
= 3 * sqrt(2) - 1
```

FYI [1,2,2,4] makes the calculation a lot easier :).